package.json

```diff
@@ -71,8 +71,7 @@
     "react-router": "5.2.1",
     "react-router-dom": "5.3.0",
     "regenerator-runtime": "0.13.8",
-    "semver": "7.3.4",
-    "typescript": "4.4.3"
```

Comment:
typescript is needed

Reply:
@mtrezza could you specify what is using typescript? I checked the repo and there is no configuration for TS, there are no `.tsx?` files and no build set up for TS 🤔

Reply:
madge uses it

Reply:
@mtrezza I checked the report `madge` generates and everything works.

<img width="712" alt="Screen Shot 2021-11-01 at 16 28 49" src="https://user-images.githubusercontent.com/920747/139697248-d74f5719-b3a9-49fe-9e93-d2a5979763a4.png">

> even if you don't see any reference for a dependency in the code base, another dependency may require it based on its configuration.

Yeah, I get that, I wouldn't try to send the PR otherwise. I checked peer deps of other packages before to be sure if TS is really not required.

Reply:
![image](https://user-images.githubusercontent.com/5673677/139709938-235c5b72-5103-4f33-8e8d-639e46ade57f.png)